src/containers/FormikForm/FormikGrepCodesContent.tsx

```diff
@@ -73,28 +80,40 @@ const FormikGrepCodesContent = ({
     });
   };
 
-  const createNewGrepCode = async (newGrepCode: string) => {
-    const grepCode = newGrepCode.toUpperCase();
-    const grepCodeTitle = await fetchGrepCodeTitle(grepCode);
-    if (
-      grepCodeTitle &&
-      !grepCodes.filter(c => c.code === grepCode).length &&
-      isGrepCodeValid(grepCode)
-    ) {
-      const temp = [
-        ...grepCodes,
-        {
+  const fetchGrepCodeTitles = async (newGrepCodes: string[]) => {
+    const newGrepCodeNames = [];
+    for (const grepCode of newGrepCodes) {
+      const grepCodeTitle = await fetchGrepCodeTitle(grepCode);
+      if (
+        grepCodeTitle &&
+        !grepCodes.filter(c => c.code === grepCode).length &&
+        isGrepCodeValid(grepCode)
+      ) {
+        newGrepCodeNames.push({
           code: grepCode,
           title: `${grepCode} - ${grepCodeTitle}`,
-        },
-      ];
-      setGrepCodes(temp);
-      updateFormik(
-        field,
-        temp.map(c => c.code),
-      );
-      form.setFieldTouched('grepCodes', true, true);
+        });
+      } else {
+        setFailedGrepCodes(true);
```

Comment:
Dersom failedGrepCodes er en [string]:
```suggestion
        setFailedGrepCodes(prevState => [...prevState, grepCode]);
```